src/ts/components/volumeslider.ts

```diff
@@ -83,44 +93,20 @@ export class VolumeSlider extends SeekBar {
     volumeChangeHandler();
   }
 
-  private detectVolumeControlAvailability(player: bitmovin.PlayerAPI): boolean {
-    // Store current player state so we can restore it later
-    let volume = player.getVolume();
-    let muted = player.isMuted();
-    let playing = player.isPlaying();
-
+  private detectVolumeControlAvailability(): boolean {
     /*
      * "On iOS devices, the audio level is always under the user’s physical control. The volume property is not
      * settable in JavaScript. Reading the volume property always returns 1."
      * https://developer.apple.com/library/content/documentation/AudioVideo/Conceptual/Using_HTML5_Audio_Video/Device-SpecificConsiderations/Device-SpecificConsiderations.html
-     *
-     * Our player API returns a volume range of [0, 100] so we need to check for 100 instead of 1.
      */
-
-    // Only if the volume is 100, there's the possibility we are on a volume-control-restricted iOS device
-    if (volume === 100) {
-      // We set the volume to zero (that's the only value that does not unmute a muted player!)
-      player.setVolume(0, VolumeSlider.issuerName);
-      // Then we check if the value is still 100
-      if (player.getVolume() === 100) {
-        // If the volume stayed at 100, we're on a volume-control-restricted device
-        return false;
-      } else {
-        // We can control volume, so we must restore the previous player state
-        player.setVolume(volume, VolumeSlider.issuerName);
-        if (muted) {
-          player.mute(VolumeSlider.issuerName);
-        }
-        if (playing) {
-          // The volume restore above pauses autoplay on mobile devices (e.g. Android) so we need to resume playback
-          // (We cannot check isPaused() here because it is not set when playback is prohibited by the mobile platform)
-          player.play(VolumeSlider.issuerName);
-        }
-        return true;
-      }
-    } else {
-      // Volume is not 100, so we're definitely not on a volume-control-restricted iOS device
-      return true;
-    }
+    // as muted autoplay gets paused as soon as we unmute it, we may not touch the volume of the actual player so we
+    // probe a dummy video element with a minimal audio source
+    const dummyVideoElement = document.createElement('video');
+    const dummyAudioSource = document.createElement('source');
```

Comment:
Setting the volume will also work without an audio source so you can remove it

Reply:
You were right, tested it today and removed the dummyAudioSource

Reply:
But the source element itself is still needed?

Reply:
Changed it to probe an `audio` element, i though it would be more safe to probe a video element with an audio source but it seems to be an overhead

Reply:
Why did you change it to an `audio` element?

Reply:
Because we want to check if we can change the audio volume?